napari/layers/points/points.py

```diff
@@ -49,6 +51,8 @@ class Points(Layer):
     properties : dict {str: array (N,)}, DataFrame
         Properties for each point. Each property should be an array of length N,
         where N is the number of points.
+    text : str, array, dict
+        Text to be displayed with the points.
```

Comment:
Can you expand here what the different options do. Would a single str get applied to all the points?

Reply:
Will do!